code/game/objects/items/weapons/storage/fancy.dm

```diff
@@ -385,6 +385,9 @@
 /obj/item/storage/lockbox/vials/AltClick(mob/user)
```

Comment:
I feel like this override shouldn't even exist? The only difference I can see is that the parent type does a raw `icon_state` assignment while this subtype calls `update_icon()`
Unless I'm missing something, I believe the `update_icon()` method is preferred, so this override can just be removed, the parent proc should call `update_icon()`, and its `update_icon_state()`/`update_overlays()` be modified as necessary

Reply:
I tried that originally, but the instead of the Lid being removed, the box got removed, leaving only the lid and vial overlay.

Reply:
Well, I assume that's where the "`update_icon_state()`/`update_overlays()` be modified as necessary" part comes in